bin/build.py

```diff
@@ -60,6 +60,8 @@
 )
 
 parser.add_argument('--test', action='store_true', help="Run ctest after build")
+parser.add_argument('--test_xml', help="Save ctest output to xml")
```

Comment:
All toolchain's name use `-` instead of `_` (also it's `--framework-device`). I think we can use same conventions here: `--test-xml`.